Demo_1/index.html

```diff
@@ -0,0 +1,496 @@
+                }
+            });
+        });
+    </script>
+
+    <style>
+        [unresolved] {
+            opacity: 0;
+        }
+
+        * {
+            font-family: sans-serif;
+        }
+
+        html, body {
+            font-size: 14px;
+            transition: opacity 0.4s;
+            margin: 0;
+            padding: 0;
+        }
+
+        main {
+            display: block;
+            transition: all 0.3s;
+        }
+
+        body {
+            overflow-y: scroll;
+            padding-top: 150px;
+        }
+
+        header {
+            width: 100%;
+            height: 150px;
+            background: #0b97c4;
+            color: white;
+            position: fixed;
+            top: 0;
+            z-index: 99991;
+        }
+
+        .sortboxContainer {
+            height: 75px;
+            position: absolute;
+            left: 15px;
+        }
+
+        .sortboxContainer .sortBoxCaption {
+            box-sizing: border-box;
+        }
+
+        nav {
+            height: 75px;
+            background-color: #23282d;
+        }
+
+        #searchHeader {
+            font-size: 34px;
+            width: 100%;
+            display: flex;
+            justify-content: center;
+        }
+
+        ff-searchbox {
+            width: 50%;
+        }
+
+        ff-searchbox input {
+            width: 100%;
+            outline: none;
+            background: #0b97c4;
+            border: none;
+            color: white;
+            margin-top: 10px;
+            height: 55px;
+            font-size: 34px;
+            line-height: 35px;
+            border-bottom: 1px solid white;
+            text-align: center;
+        }
+
+        ff-searchbutton {
+            display: flex;
+        }
+
+        button {
+            cursor: pointer;
+            background: none;
+            outline: none !important;
+            border: none;
+        }
+
+        ff-campaign-feedbacktext {
+            width: 100%;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            flex-direction: row;
+        }
+
+        #rightColumn {
+            display: flex;
+            justify-content: center;
+            flex-direction: column;
+        }
+
+        span.rating {
+            color: gold;
+            font-size: 1.5rem;
+        }
+    </style>
+</head>
+<body>
+<ff-campaign-redirect></ff-campaign-redirect>
+<ff-communication url="https://search-web-components.fact-finder.de/FACT-Finder-7.2"
+                  version="7.2"
+                  channel="bergfreunde-co-uk"
+                  default-query="sale"
+                  currency-country-code="de-DE"
+                  currency-code="EUR"
+                  search-immediate></ff-communication>
+
+<header>
+    <nav>
+        <ff-header-navigation group-count="4" unresolved>
+            <img slot="container-left-Outdoor clothing" src="images/outdoorClothing.jpg" style="max-width: 1200px"
```

Comment:
There's too much inline CSS in this file. It's everywhere. Maybe it's high time to introduce some classes?

Reply:
I think no one of us would use such much inline styles today. But as I expect we are not really going to touch the old demos ever again not worth the effort